test/Feature/Atomic.c

```diff
@@ -0,0 +1,69 @@
+// REQUIRES: geq-llvm-3.7
+// RUN: %llvmgcc %s -emit-llvm -S -c -o - | FileCheck %s
```

Comment:
Great! That's a good approach to do the checking of assembly. Still, what might happen to you is that KLEE optimises it away. Could you first run KLEE and after that check the `%t.klee-out/assembly.ll`. In this case we know that KLEE saw it.